pytorch_lightning/utilities/distributed.py

```diff
@@ -203,10 +204,13 @@ def all_gather_ddp_if_available(
     Return:
         A tensor of shape (world_size, batch, ...)
     """
+    if group is None:
+        group = torch.distributed.group.WORLD
+
     if torch.distributed.is_available() and torch.distributed.is_initialized():
         if sync_grads:
             return AllGatherGrad.apply(tensor, group)
         else:
-            with torch.no_grad:
+            with torch.no_grad():
```

Comment:
wasn't this already merged?

Reply:
Yes, it was but on master. This new release system might be problematic in the future. Let's see how it goes.